index.html

```diff
@@ -1478,21 +1465,26 @@ <h2>The <dfn>NFCRecordType</dfn> enum</h2>
     </thead>
     <tbody>
      <tr>
-      <td>[[\suspended]]</td>
+      <td>[[\Suspended]]</td>
       <td>
         A boolean flag indicating whether NFC functionality is
         <a href="#nfc-suspended">suspended</a> or not, initially
         <code>false</code>.
       </td>
      </tr>
      <tr>
-      <td>[[\watchList]]</td>
+      <td>[[\WatchList]]</td>
       <td>
         A <a>set</a> of <a>NFC watch</a>es initially set to the empty <a>set</a>.
       </td>
      </tr>
     </tbody>
   </table>
+  <p class="note">
+    Internal slots are used only as a notation in this specification, and
+    implementations do not necessarily have to map them to explicit internal
+    properties.
+  </p>
```

Comment:
Great note

Reply:
(It was there before; I just moved it.)

Reply:
I was wondering, should we add a note that there's only one NFC device per UA?

Reply:
There could potentially be multiple, it would just push a write to both in parallel, as well as set up the same watch for both